sqlglot/dialects/duckdb.py

```diff
@@ -215,15 +221,8 @@ class Parser(parser.Parser):
             "XOR": binary_from_function(exp.BitwiseXor),
         }
 
-        FUNCTION_PARSERS = {
-            **parser.Parser.FUNCTION_PARSERS,
-            "DECODE": lambda self: self.expression(
-                exp.Decode, this=self._parse_conjunction(), charset=exp.Literal.string("utf-8")
-            ),
-            "ENCODE": lambda self: self.expression(
-                exp.Encode, this=self._parse_conjunction(), charset=exp.Literal.string("utf-8")
-            ),
-        }
+        FUNCTION_PARSERS = parser.Parser.FUNCTION_PARSERS.copy()
+        FUNCTION_PARSERS.pop("DECODE", None)
```

Comment:
We have `"DECODE"` in the base parser's `FUNCTION_PARSERS` which would take precedence over `FUNCTIONS`. This wasn't previously an issue because DuckDB overrode `"DECODE"` so we'd hit the correct logic.